packages/atomic/cypress/integration/facets/color-facet/color-facet-selectors.ts

```diff
@@ -18,18 +18,22 @@ export const ColorFacetSelectors = {
     return this.shadow().find('[part="placeholder"]');
   },
   selectedBoxValue() {
-    return this.shadow().find('[part="value-box"][aria-pressed="true"]');
+    return this.shadow().find(
+      '[part~="value-box"][part~="value-box-selected"][aria-pressed="true"]'
```

Comment:
This selector is redundant, but it also garantees that aria-pressed and -selected are present at the same time.